src/kv/store.h

```diff
@@ -638,7 +638,6 @@ namespace kv
 
         version = tx_id.version;
         last_replicated = tx_id.version;
-        last_committable = 0;
```

Comment:
The comment on lines `50-51` needs updating.

Reply:
Good catch, thank you!